File: Lab2/lab2.py
print("Please enter x and y: ")

x = float(input("x = "))
y = float(input("y = "))

if (x > 0 and y > 0):
    print("I quadrant")
elif (x < 0 and y > 0):
    print("II quadrant")
elif (x < 0 and y < 0):
    print("III quadrant")
elif (x > 0 and y < 0):
    print("IV quadrant")
elif (x == 0, y == 0):
    print("It's origin")

File: Lab2/lab2.cpp
#include <iostream>
using namespace std;

int main()
{
    double x, y;
    cout << "Please enter x and y: ";
    cout << "x = ";
    cin >> x;
    cout << "y = ";
    cin >> y;
    if (x > 0 && y > 0)
        cout << "I quadrant"
             << "\n";
    else if (x < 0 && y > 0)
        cout << "II quadrant"
             << "\n";
    else if (x < 0 && y < 0)
        cout << "III quadrant"
             << "\n";
    else if (x > 0 && y < 0)
        cout << "IV quadrant"
             << "\n";
    else if (x == 0, y == 0)
        cout << "It's origin"
             << "\n";
    return (0);
}